service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -50,11 +52,29 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         if ("FPLA-2544".equals(migrationId)) {
             run2544(caseDetails);
         }
+        if ("FPLA-2481".equals(migrationId)) {
+            run2481(caseDetails);
+        }
 
         caseDetails.getData().remove(MIGRATION_ID_KEY);
         return respond(caseDetails);
     }
 
+    private void run2481(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+
+        if ("LE20C50023".equals(caseData.getFamilyManCaseNumber())) {
+            Others others = caseData.getOthers();
+
+            if (isEmpty(others.getAdditionalOthers())) {
+                caseDetails.getData().remove("others");
+            } else {
+                others = others.toBuilder().firstOther(null).build();
```

Comment:
this does not look right to me. probably first additional other should become first other